client/main.lua

```diff
@@ -905,7 +909,6 @@ Citizen.CreateThread(function()
 		if ESX.PlayerData.job and ESX.PlayerData.job.name == 'mechanic' then
 			local coords, letSleep = GetEntityCoords(PlayerPedId()), true
 			local isInMarker, hasExited = false, false
-			local currentStation, currentPart, currentPartNum
```

Comment:
This variables are not initialized on the current update. This could become a potential bug and if multiple stations are set in the config.